package.json

```diff
@@ -52,35 +54,35 @@
     "code": "2.x.x",
     "cookie": "0.2.x",
     "del": "2.x.x",
-    "eslint-plugin-hapi": "4.x.x",
+    "eslint-plugin-hapi": "^4.x.x",
+    "eslint-plugin-react": "^5.0.1",
     "flux": "2.x.x",
     "flux-constant": "1.x.x",
     "flux-dispatcher": "1.x.x",
     "flux-store": "1.x.x",
     "font-awesome": "4.x.x",
-    "gulp": "3.x.x",
-    "gulp-concat": "2.x.x",
-    "gulp-less": "3.x.x",
-    "gulp-newer": "1.x.x",
-    "gulp-nodemon": "2.x.x",
-    "gulp-rename": "1.x.x",
-    "gulp-sourcemaps": "1.x.x",
-    "gulp-util": "3.x.x",
-    "history": "1.17.x",
-    "jsdom": "7.x.x",
-    "lab": "8.x.x",
+    "gulp": "^3.x.x",
+    "gulp-concat": "^2.x.x",
+    "gulp-less": "^3.x.x",
+    "gulp-newer": "^1.x.x",
+    "gulp-nodemon": "^2.x.x",
+    "gulp-rename": "^1.x.x",
+    "gulp-sourcemaps": "^1.x.x",
+    "gulp-util": "^3.x.x",
+    "history": "^1.17.x",
+    "jsdom": "^7.x.x",
+    "lab": "^8.x.x",
     "lab-babel": "1.1.1",
     "lodash": "3.x.x",
     "merge-stream": "1.x.x",
     "moment": "2.x.x",
-    "nodemon": "1.x.x",
+    "nodemon": "^1.x.x",
     "promptly": "0.2.x",
     "proxyquire": "1.x.x",
     "qs": "6.x.x",
     "react-addons-linked-state-mixin": "0.14.x",
     "react-addons-test-utils": "0.14.3",
-    "react-dom": "0.14.x",
-    "react-router": "1.x.x",
+    "react-router": "^1.x.x",
```

Comment:
There is React 15; [https://github.com/facebook/react/releases](https://github.com/facebook/react/releases)